Service_Message_WithOrgLogo/index.html

```diff
@@ -326,32 +328,29 @@
-                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:400;line-height:24px;text-align:left;color:#5C6F82;">Divisione servizi<br /> Carta Giovani Nazionale</div>
+                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:400;line-height:24px;text-align:left;color:#5C6F82;">${senderMetadata.departmentName}<br /> ${senderMetadata.serviceName}</div>
                                 </td>
                               </tr>
+                              <!-- Title & Main Message -->
                               <tr>
                                 <td align="left" class="title" style="font-size:0px;padding:10px 25px;padding-top:24px;padding-bottom:0px;word-break:break-word;">
-                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:13px;font-weight:bold;line-height:1;text-align:left;color:#17324D;">
-                                    <!-- H4 Desktop (from MUI Italia)-->
-                                    <h1 style="font-size: 32px; line-height: 40px; margin: 0"> La tua Carta Giovani è attiva! </h1>
+                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:13px;font-weight:bold;line-height:1;text-align:left;color:#17324D;"><!-- H4 Desktop (from MUI Italia)-->
+                                    <h1 style="font-size: 32px; line-height: 40px; margin: 0"> ${subject} </h1>
                                   </div>
                                 </td>
                               </tr>
                               <tr>
                                 <td align="left" class="text" style="font-size:0px;padding:10px 25px;padding-top:4px;word-break:break-word;">
-                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:regular;line-height:24px;text-align:left;color:#17324D;">
-                                    <!-- Body 1 Desktop (from MUI Italia)-->
-                                    <p> Buone notizie! <strong>La tua Carta Giovani Nazionale è attiva</strong> e la potrai trovare all’interno della sezione Portafoglio. </p>
-                                    <p> Ti ricordiamo che solo tu puoi usufruire degli sconti presentando la Carta presso gli esercenti fisici aderenti, oppure inserendo i codici nell’apposito campo presso gli esercenti aderenti online. </p>
-                                    <p> La Carta Giovani Nazionale sarà valida da oggi fino al compimento dei 36 anni. Inizia subito a usarla! </p>
-                                  </div>
+                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:regular;line-height:24px;text-align:left;color:#17324D;"><!-- Body 1 Desktop (from MUI Italia)--> ${body}</div>
```

Comment:
We should make sure that the `${body}` placeholder is replaced with the correct paragraph tags. For example, if we have two different paragraphs:
```html
<p>First paragraph</p>
<p>Second paragraph</p>
```
This is essential if we want the mail to render correctly on different email clients.

Reply:
this option is not feasable due to client compatibility